frontend/src/components/project_assignment/assignmentItem.tsx

```diff
@@ -0,0 +1,93 @@
+import { IProjectSkill } from "../../api/entities/ProjectSkillEntity";
+import { Badge, CloseButton, Col, Row } from "react-bootstrap";
+import useSWR, { useSWRConfig } from "swr";
+import { deleteAssignment, getAssignments } from "../../api/calls/AssignmentCalls";
+import WarningToast from "../warningToast";
+import apiPaths from "../../properties/apiPaths";
+import useTranslation from "next-translate/useTranslation";
+import { capitalize } from "../../utility/stringUtil";
+import { getSkillTypeFromSkill } from "../../api/calls/skillTypeCalls";
+import { useEffect, useState } from "react";
+
+/**
+ * An item containing the information about the assignments to a skill of a project.
+ * It contains the name of the skill followed by a list of assignments for that skill.
+ * The list contains the first name of the student, the name of the person that did the assignment and the motivation.
+ * Furthermore, it contains a delete button to delete the assignment.
+ * @param item Skill you want the assignments from
+ * @constructor
+ */
+function AssignmentItem(item: { skill: IProjectSkill }) {
+    const { t } = useTranslation("common");
+
+    const [color, setColor] = useState("secondary");
+    useEffect(() => {
+        getSkillTypeFromSkill(item.skill).then((skillType) => setColor(skillType.colour));
+    });
+
+    const { mutate } = useSWRConfig();
+    let { data, error } = useSWR(item.skill._links.assignments.href, getAssignments, {
+        refreshInterval: 10,
```

Comment:
I think this is the default value so this should not be needed? Also, this value is way to small? IRC this is in ms?